infrastructure/main.tf

```diff
@@ -103,6 +104,7 @@ module "api-gateway-web" {
     PROXY_DATA = "http://ccd-data-store-api-${local.env_ase_url}"
     PROXY_DEFINITION_IMPORT = "http://ccd-definition-store-api-${local.env_ase_url}"
     PROXY_DOCUMENT_MANAGEMENT = "${local.document_management_url}"
+    PROXY_MV_ANNOTATIONS_API_URL = "${local.mv_annotations_api_url}"
```

Comment:
is the declaration of a local var needed? 
Can you try removing line 22 and here directly with:
```
PROXY_MV_ANNOTATIONS_API_URL = "${var.mv_annotations_api_url}"
```

Reply:
Fixed.